src/Committee/CommitteeManager.php

```diff
@@ -366,6 +377,51 @@ public function unfollowCommittee(Adherent $adherent, Committee $committee): voi
         }
     }
 
+    /**
+     * Makes an adherent vote in one committee.
+     *
+     * @param Adherent  $adherent  The follower
+     * @param Committee $committee The committee to follow
+     */
+    public function enableVoteInCommittee(Adherent $adherent, Committee $committee): ?CommitteeMembership
+    {
+$membership = $this->getCommitteeMembership($adherent, $committee);
+
+        if ($membership) {
+            if ($existingMembership = $this->getMembershipRepository()->findVotingMembership($adherent)) {
+                $existingMembership->disableVote();
+            }
+            $membership->enableVote();
+
+            $this->getManager()->flush();
+
+            return $membership;
```

Comment:
you should move this `return` statement outside the IF, otherwise, your method returns nothing in case if any membership was found